### PR TITLE
pandora_server: add support to accept bumble config file

### PR DESCRIPTION
**Pull request by SilverBzH**

### CODE REVIEW

File: apps/pandora_server.py [hunk 1]
```diff
@@ -18,13 +20,30 @@
...
+    if 'transport' not in bumble_config.keys():
+        bumble_config.update({'transport': transport})
+    device = PandoraDevice(bumble_config)
+
     logging.basicConfig(level=logging.DEBUG)
     asyncio.run(serve(device, port=grpc_port))
 
 
+def retrieve_config(config: str) -> Dict[str, Any]:
+    if len(config) == 0:
```
Comment (barbibulle) [inline apps/pandora_server.py:41]: recommend: using "implicit" false test (https://google.github.io/styleguide/pyguide.html#214-truefalse-evaluations)

File: apps/pandora_server.py [hunk 2]
```diff
@@ -18,13 +20,30 @@
...
+        return {}
+    else:
```
Comment (barbibulle) [inline apps/pandora_server.py:43]: `else` not needed here.

**Review by barbibulle — APPROVED** [review]

File: apps/pandora_server.py (hunk 1 above)
Comment (SilverBzH) [reply apps/pandora_server.py]: Indeed this is cleaner, thanks !

File: apps/pandora_server.py (hunk 2 above)
Comment (SilverBzH) [reply apps/pandora_server.py]: Of course 👍

**Pull request merged**